client-participation/package.json

```diff
@@ -61,7 +74,7 @@
     "gulp-markdown": "~0.1.2",
     "gulp-minify-css": "~0.3.0",
     "gulp-rename": "~1.0.0",
-    "gulp-s3": "git://github.com/mbjorkegren/gulp-s3.git#847e2b372204b4ceeccb0e9b567f28ff00e7fa1d",
+    "gulp-s3": "github:pol-is/gulp-s3#master",
```

Comment:
curious, why do we fork `gulp-s3`. fork here: https://github.com/pol-is/gulp-s3

Reply:
github tells me 

> This branch is even with mbjorkegren:master.

maybe I am missing something.

Reply:
ok, I think this is the answer https://github.com/pol-is/polis/pull/743

Reply:
The original pinned commit was from a contributor's personal fork, not the original library author. makes sense to have an official polis fork then. carry on.

Reply:
hm. not sure. i usually consider all the stuff from past out of scope for my PRs, unless i'm tackling just that

But looks like this is the scope of change -- creating a new path in bucket:
https://github.com/pol-is/gulp-s3/compare/1c4f9ff19472a4e2584f3f85d0c1274a7963fef3...master

I'm supportive of someone tracking these down and reverting back to mainline tho! Should it be a future PR thing?